app/assets/javascripts/client/common_create/ToolDetailsCtrl.js

```diff
@@ -37,7 +37,11 @@
 
     vm.save = function(entity) {
       CreateService.loadDistrict(vm.district);
-      CreateService.saveAssessment(entity);
+      if(CreateService.context === 'inventory') {
```

Comment:
I think this functionality is envying `CreateService`, if we pass the entity to `CreateService.save` then the service should be able to figure it out.